@xen-orchestra/web-core/lib/components/ui/alarm-item/UiAlarmItem.vue

```diff
@@ -0,0 +1,142 @@
+<!-- v1 -->
+<template>
+  <li class="ui-alarm-item" :class="className">
+    <div class="content">
+      <div class="title-progress text-ellipsis">
+        <UiButtonIcon
+          v-if="alarm.description"
+          :icon="isDescriptionVisible ? faAngleDown : faAngleRight"
+          size="small"
+          accent="brand"
+          :target-scale="2"
+          @click="toggleDescription()"
+        />
+        <span v-tooltip class="typo-body-regular text-ellipsis">
+          {{ alarm.label }}
+        </span>
+        <span class="typo-body-regular value">
+          {{ alarm.value }}
+        </span>
+      </div>
+      <div class="typo-body-regular-small info">
+        <div v-if="slots.link" class="object-link">
+          {{ t('on-object') }}
+          <span class="descriptor">
+            <slot name="link" />
+          </span>
+          <span class="interpunct" />
+        </div>
+        <span class="typo-body-regular-small">
+          {{ timeAgo }}
+        </span>
+      </div>
+    </div>
+    <div v-if="isDescriptionVisible" class="description">
+      {{ alarm.description }}
+    </div>
+  </li>
+</template>
+
+<script lang="ts" setup>
+import { vTooltip } from '@core/directives/tooltip.directive'
+import { toVariants } from '@core/utils/to-variants.util'
+import { faAngleDown, faAngleRight } from '@fortawesome/free-solid-svg-icons'
+import { useTimeAgo, useToggle } from '@vueuse/core'
+import { computed } from 'vue'
+import { useI18n } from 'vue-i18n'
+import UiButtonIcon from '../button-icon/UiButtonIcon.vue'
+
+type Alarm = {
+  label: string
+  value: string
+  date: Date | number | string
+  description?: string
+}
+
+const { alarm, size } = defineProps<{
+  alarm: Alarm
+  size: 'small' | 'large'
+}>()
+
+const slots = defineSlots<{
+  link?(): any
+}>()
+
+const timeAgo = useTimeAgo(alarm.date)
```

Comment:
Beware of loss of reactivity here

```suggestion
const timeAgo = useTimeAgo(() => alarm.date)
```